### DISCUSSION TITLE
V2.0 info (read this first!)

### DISCUSSION THREAD

**Discussion by jeroenterheerdt** [jeroenterheerdt/HAsmartirrigation#240] [Announcements]
Please read the [wiki ](https://github.com/jeroenterheerdt/HAsmartirrigation/wiki) and [readme](https://github.com/jeroenterheerdt/HAsmartirrigation/blob/v2.0/README.md). If you have questions after this start a discussion or if you have encountered a bug or have a feature request open an issue.

**Comment by gatonero** [comment]
Good idea to start discussions!
